packages/gatsby-plugin-no-sourcemaps/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "gatsby-plugin-no-sourcemaps",
+  "version": "0.0.1",
+  "description": "Disable sourcemaps when building javascript",
+  "main": "index.js",
+  "keywords": [
+    "gatsby",
+    "webpack",
+    "sourcemaps"
+  ],
+  "author": "Stuart taylor <stuart@freecodecamp.org>",
+  "license": "BSD-3-Clause"
```

Comment:
Any reason to use this over MIT? All our other packages are MIT so it's more confusing to have multiple licenses in the repo.

Reply:
The only reason is that it's what I normally work to. I can change to MIT though. No issues